### DISCUSSION TITLE
How to update the theme to latest version

### DISCUSSION THREAD

**Discussion by ExtremeKrish** [cotes2020/jekyll-theme-chirpy#1439] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I've generated my website through chirpy starter and now I want to update the theme version,how can I update to the latest version  whenever the new version comes?

### Operations you have already tried

Nothing...

**Comment by joshjohanning** [comment, marked as answer]
From the starter is easy, just update the `Gemfile`

https://github.com/cotes2020/jekyll-theme-chirpy/wiki/Upgrade-Guide#upgrade-from-starter

**Reply by zhang-nianqiang** [reply]
I'm a newbie. I'm having the same issue.
Like the 6.4 version just released, I modified the Gemfile but something went wrong.
1. The compilation hint has an extra "/" before the image URL(I didn't set anything up in the CDN section.)(It was fine before)
2. Will the assets folder be automatically upgraded?

**Reply by joshjohanning** [reply]
I use the fork method so I'm not 100% sure, but after updating the `gemfile`, I believe you run `bundle update jekyll-theme-chirpy` and commit the new files.

**Reply by zhang-nianqiang** [reply]
Thank you very much for your reply.
My computer doesn't have a compilation environment.
Maybe I should configure the compilation environment on my computer.
Thanks again for your reply. 👍

**Reply by joshjohanning** [reply]
You do get 60 hours of [free Codespaces compute a month](https://github.com/features/codespaces), so you could develop your blog in that (that's what I primarily use to write blog posts, update the theme, etc.)

**Reply by zhang-nianqiang** [reply]
Thanks for the suggestion.
I tried to use it but my internet speed and laptop were too slow.
While I was waiting, I had a new idea.I feel like it's great.
When there is an update to the theme, use the `chirpy-starter` for redeployment.
Since I haven't made a lot of changes to the theme, redeploying is more suitable for me. 👍
